### PR TITLE
Add About wercker

### PR DESCRIPTION
**Pull request by Tonnu**
Provides a high-level overview of what wercker does and also a list of terminologies used throughout the docs.

Also a minor (style) fix to index.md

### CODE REVIEW

File: content/docs/about.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+| Pipeline | A sequential execution of Steps of which the end result can either pass or fail. |
+| Registry | The marketplace that contains all the steps developed by the community. |
+| Service | A container that is spun up separate from you main pipeline and runs something like a database, message queue that you need for your Pipelines to run. Note that a service could also be another application that you've created on wercker. |
+| Stack | The underlying infrastructure that runs your Pipelines. There are currently two versions. Ewok (Docker-based) and the newest. Classic (Andorian) which runs LXC containers and will be deprecated. |
+| Step | Steps are individual units of work, they can pass or fail, and a failing step will prevent the execution of a step after it.  There are three flavors of steps: <ul><li> 1. An external step from the Registry, created by wercker itself or the community. It comes in the form of $username/$step_name </li><li> 2. Script step, an inline piece of code that gets turned into a run.sh on the fly </li><li> 3. Internal Step implemented by the wercker CLI that runs with enhanced privileges that aren't available in the container </li></ul>|
```
Comment (hatchan) [inline content/docs/about.md]: is it possible to get rid of the html tags in here?

File: content/docs/about/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+At wercker, we believe that micro-services are the future. However, with great
+power comes great responsibility and micro-services naturally introduce a lot
+of extra moving parts in your application. To fully leverage the advantages of
+micro-services, they should be managed carefully.
+
+Wercker lets you do just that. By setting up automated development pipelines,
+very much in the spirit of "_release early, release often_", you can build and
+deploy your services with just a `git push`. You can then let wercker compile
+and run any other _steps_ that are necessary to build your project before it
+gets deployed to a [target](http://devcenter.wercker.com/docs/deploy/deploy-steps.html)
```
Comment (hatchan) [inline content/docs/about/index.md]: link doesn't work, also probably a better idea to leave out the host: `/docs/deploy/deploy-steps.html`

File: content/docs/about/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+| Environment Variable | A key value pair that holds a specific set of information such as a token, username or SSH key. Environment Variables can hold plain text (username), be protected (hidden, for instance for passwords) or hold an SSH key. Environment Variables look as follows: USERNAME=mies and be leveraged by referencing it as follows: $USERNAME. Environment Variables can be assigned to either an entire Pipeline or a Deploy Target. |
+| Parameters | The arguments and options you pass along to a Step. |
+| Pipeline | A sequential execution of Steps of which the end result can either pass or fail. |
+| Registry | The marketplace that contains all the steps developed by the community. |
+| Service | A container that is spun up separate from you main pipeline and runs something like a database, message queue that you need for your Pipelines to run. Note that a service could also be another application that you've created on wercker. |
+| Stack | The underlying infrastructure that runs your Pipelines. There are currently two versions. Ewok (Docker-based) and the newest. Classic (Andorian) which runs LXC containers and will be deprecated. |
```
Comment (hatchan) [inline content/docs/about/index.md]: Talked a bit offline about this, but we probably want to discuss wether we're going to use the name ewok or not

Comment (hatchan) [inline content/docs/about/index.md]: Also, we need to switch the items in `Ewok (Docker-based)`, so they match with `Classic (Andorian)` (or vice versa)

File: content/docs/about/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+| Environment | An entire set of Environment Variables. | 
+| Environment Variable | A key value pair that holds a specific set of information such as a token, username or SSH key. Environment Variables can hold plain text (username), be protected (hidden, for instance for passwords) or hold an SSH key. Environment Variables look as follows: USERNAME=mies and be leveraged by referencing it as follows: $USERNAME. Environment Variables can be assigned to either an entire Pipeline or a Deploy Target. |
+| Parameters | The arguments and options you pass along to a Step. |
+| Pipeline | A sequential execution of Steps of which the end result can either pass or fail. |
+| Registry | The marketplace that contains all the steps developed by the community. |
+| Service | A container that is spun up separate from you main pipeline and runs something like a database, message queue that you need for your Pipelines to run. Note that a service could also be another application that you've created on wercker. |
```
Comment (hatchan) [inline content/docs/about/index.md]: Currently we do not support the thing that is mentioned in the `note` part. So probably remove it for now

**Comment by hatchan** [comment]
Small things, lgtm (note, one of my comments was made into a outdated comment, but it is still applicable)

File: content/docs/about.md (hunk 1 above)
Comment (Tonnu) [inline content/docs/about.md]: Didn't find any relevant documentation supporting lists inside tables, so used html as explained here: http://stackoverflow.com/questions/19950648/how-to-write-lists-inside-a-markdown-table

**Comment by mies** [comment]
looks good, changed about a bit. We prolly want to have another hard look at this page at a later stage.

**Pull request merged**